pipelined/bevy_render2/src/render_graph/graph.rs

```diff
@@ -176,8 +228,39 @@ impl RenderGraph {
         Ok(())
     }
 
-    pub fn validate_edge(&mut self, edge: &Edge) -> Result<(), RenderGraphError> {
-        if self.has_edge(edge) {
+    pub fn remove_node_edge(
+        &mut self,
+        output_node: impl Into<NodeLabel>,
+        input_node: impl Into<NodeLabel>,
+    ) -> Result<(), RenderGraphError> {
+        let output_node_id = self.get_node_id(output_node)?;
+        let input_node_id = self.get_node_id(input_node)?;
+
+        let edge = Edge::NodeEdge {
+            output_node: output_node_id,
+            input_node: input_node_id,
+        };
+
+        self.validate_edge(&edge, true)?;
+
+        {
+            let output_node = self.get_node_state_mut(output_node_id)?;
+            output_node.edges.remove_output_edge(edge.clone())?;
+        }
+        let input_node = self.get_node_state_mut(input_node_id)?;
+        input_node.edges.remove_input_edge(edge)?;
+
+        Ok(())
+    }
+
+    pub fn validate_edge(
+        &mut self,
+        edge: &Edge,
+        should_exist: bool,
+    ) -> Result<(), RenderGraphError> {
```

Comment:
The `bool` argument here is tough to grok at the call site:

```rust
validate_edge(&edge, false);
```

It's not clear what `false` means unless you go look at the docs and read the argument name. (See [C-CUSTOM-TYPE](https://rust-lang.github.io/api-guidelines/type-safety.html#arguments-convey-meaning-through-types-not-bool-or-option-c-custom-type) in the Rust API guidelines.)

Something like

```rust
enum EdgeExists {
    Yes, // Or `True`, or `Exists`
    No,  // Or `False`, or `DoesNotExist`
}
```

would be more explicit.

Reply:
Done.